maintainers/maintainer-list.nix

```diff
@@ -7530,6 +7530,12 @@
     githubId = 1665818;
     name = "Philipp Eder";
   };
+  nickcao = {
+    name = "Nick Cao";
+    email = "nickcao@nichi.co";
+    github = "NickCao";
+    githubId = 15247171;
+  };
```

Comment:
This should be in a seperate commit with the message `maintainers: add <name>`. Make sure that the commit comes before the package.